src/foam/util/Emails/EmailsUtility.java

```diff
@@ -55,6 +55,7 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
     AppConfig appConfig = (AppConfig) x.get("appConfig");
     if ( user != null ) {
       userX = x.put("subject", new Subject.Builder(x).setUser(user).build());
+      userX = x.put("group", user.findGroup(x));
```

Comment:
This and the next two lines, can we just lookup group once?
Group group = user.findGroup(x),  then use it for getAppConfig and and the userX setup?

Reply:
Updated